src/components/FileItem.tsx

```diff
@@ -182,7 +190,7 @@ export class FileItem extends React.PureComponent<IFileItemProps> {
         }
         onDoubleClick={this.props.onDoubleClick}
         style={this.props.style}
-        title={this.props.trans.__(`%1 ● ${status}`, this.props.file.to)}
+        title={this.props.trans.__(`%1 • ${status}`, this.props.file.to)}
```

Comment:
What is the rationale to change that character?

Reply:
Nothing super important really,

I found that VSCode uses the smaller bullet

![image](https://user-images.githubusercontent.com/59669957/127161485-9f3e1f06-bf3b-447b-8bd5-e461a99b45a7.png)

and I preferred how that looked :sweat_smile:, quite subjective lol